bibliography/mri-analysis.md

```diff
@@ -0,0 +1,5 @@
+# MRI Analysis
+
+## Fundamentals
+
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.drcmr.dk/bloch](http://www.neuroimagingprimers.org) -- physical copy of the book coming up in Lab soon!
```

Comment:
markdown hyperlink is incorrect-- you can verify by either clicking on 'view' on GH, or building local site (https://github.com/neuropoly/intranet.neuro.polymtl.ca/wiki#build-website-locally)

Reply:
I have updated the markdown hyperlink.